Source/FunctionMonkey/ConfigurationLocator.cs

```diff
@@ -40,7 +40,7 @@ public static IFunctionAppConfiguration FindConfiguration()
 
         private static bool Scan(out MethodInfo linkBackInfo, out IFunctionAppConfiguration findConfiguration)
         {
-            Assembly[] assemblies = AppDomain.CurrentDomain.GetAssemblies();
+            Assembly[] assemblies = AppDomain.CurrentDomain.GetAssemblies().Where(a => !a.GetName().Name.StartsWith("Microsoft.")).ToArray();
```

Comment:
Is it safe to further qualify the Microsoft. prefix - Microsoft.VisualStudio perhaps?